### ISSUE TITLE
Change user roles for Units

### ISSUE THREAD

**Issue by zwaters** [kirstykitto/CLAtoolkit#47]
We are wanting to transition from simple Staff vs. Students for course offerings to allow for independently managed unit offerings as opposed to the current portal where staff can access all unit offerings.

**Comment by kirstykitto** [comment]
This does not actually seem to be an issue to me. Staff see whatever is configured for them in the django interface. Thus - they get access to all the units for which they have been added right? Or do you mean a scenario where a user can have different roles across two units? (We do want that.)

**Comment by tommarmstrong** [comment]
"Staff" users can currently edit everything instead of just courses they should be able to manage (ones that they have created). There's code that changes this in PR #55 to allow the ability to have different roles across two units like you mentioned @kirstykitto

**Comment by zwaters** [comment]
Yes, sorry - we are maintaining our current Staff/Student permissions. We want to build ontop of that so a staff user can maintain and administer/manage their own courses within the CLA Toolkit interface. This will also be helpful for a use case where a student/learner (could be anyone, really) wants to create their own module to track their learning related social media interactions (for MOOC learners, e.g.).

**Comment by kirstykitto** [comment]
Ah - yes. Completely agree - that should not happen :)
